daemon/firewall/iptables/iptables_test.go

```diff
@@ -874,13 +806,8 @@ func TestRuleToIPTables(t *testing.T) {
 				countingOnly = true
 				assert.Equal(t, tt.ipv4Count, len(allRules[ipv4Table]))
 			}
-			if tt.ipv6Count > 0 {
-				countingOnly = true
-				assert.Equal(t, tt.ipv6Count, len(allRules[ipv6Table]))
-			}
 			if countingOnly == false {
```

Comment:
This could probably be simplified by having condition only on `tt.ipv4Count` now when we don't have `ipv6Count`